draft-meynell-panrg-scion-research-questions.md

```diff
@@ -200,6 +201,33 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 * One AS belongs to many ISDs?
 
 
+## DNS Service Binding (SVCB)
+
+The DNS Service Binding {{?RFC9460, Section 14.3}} allows a dedicated SCION Service Parameter to be specified.
+
+Service Parameters allow the specification of alternative IP addresses or other parameters such as ISD/AS numbers) for a given URL.
+This would be more elegant than using DNS TXT records.
+
+Example of current entry:
+
+    $ dig +short ethz.ch TXT  | grep scion
+    "x-sciondiscovery=8041"
+    "scion=64-2:0:9,129.132.230.98"
+
+
+With SVCB this may look like this for https:
+
+    dig +short https ethz.ch
+    1 . alpn="h2" scion=64-2:0:9,129.132.230.98
+
+I am not entirely sure how it looks like for non-https traffic, I assume probably like this:
+
+    dig +short svcb ethz.ch
+    1 . scion=64-2:0:9,129.132.230.98
```

Comment:
Maybe you can just omit these lines if you are not sure? 
I would also not use "I" in a draft
```suggestion
```